### PR TITLE
project proposal

### PR DESCRIPTION
**Pull request by liuallen871219**
---
A good PR should include the following parts:

- [ ] A clear title
- [ ] A meaningful message for PR, as well as its commits
- [ ] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [ ] excluding any irrelevant files, such as binaries, text files, or dot files
- [ ] passing tests/CI

### CODE REVIEW

File: project/liuallen871219/README.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Iterative Closest Point
```
Comment (yungyuc) [inline project/liuallen871219/README.rst:1]: Your project does not render correctly.  You are using markdown but the file is named as a rst.

File: project/liuallen871219/README.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Problem to Solve
+Iterative closest point (ICP)is an algorithm employed to minimize the difference between two clouds of points. ICP is often used to reconstruct 2D or 3D surfaces from different scans, to localize robots and achieve optimal path planning (especially when wheel odometry is unreliable due to slippery terrain), to co-register bone models, etc.
+
+## Prospective Users
+The users who want to find transformation between two point sets. If the users have serise of rgb and depth image, they can use the ICP to do the 3D reconstruction of scene.
+## System Architecture
+![](https://i.imgur.com/uJfzY04.png)
+
+## API Description
+### Python
```
Comment (yungyuc) [inline project/liuallen871219/README.rst:14]: It's good to separate Python and C++ API.

**Review by yungyuc — CHANGES_REQUESTED** [review]
The contents are OK.  But the branch has conflict.  Please resolve the conflict and leave a comment to let me know.

File: project/liuallen871219/README.rst (hunk 1 above)
Comment (liuallen871219) [reply project/liuallen871219/README.rst:1]: I fix the conflict and change the file extension of README

Comment (yungyuc) [reply project/liuallen871219/README.rst]: You have not resolved the conflict.  Please observe your PR changeset history.  There are a lot of unrelated changesets. You cannot have them in this PR.  Please read git doc to learn how to handle it.

**Pull request closed without merging**